### PR TITLE
fix: Adding underlying library version to library field

### PR DESCRIPTION
**Pull request by izaaz**
Setting min iOS SDK to 1.6 and appending underlying SDK library to library version

### CODE REVIEW

**Review by crleona — APPROVED** [review]

File: android/src/main/kotlin/com/amplitude/amplitude_flutter/FlutterLibraryPlugin.kt [hunk 1]
```diff
@@ -9,7 +9,11 @@ class FlutterLibraryPlugin(val library: String): Plugin {
     override lateinit var amplitude: Amplitude
 
     override fun execute(event: BaseEvent): BaseEvent? {
-        event.library = library
+        if (event.library == null) {
```
Comment (Mercy811) [inline android/src/main/kotlin/com/amplitude/amplitude_flutter/FlutterLibraryPlugin.kt:12]: Shouldn't here be `!= null`?

Comment (izaaz) [reply android/src/main/kotlin/com/amplitude/amplitude_flutter/FlutterLibraryPlugin.kt:12]: We can only merge the `event.library` value if its not null. If its null, we just want to use the flutter library version `Amplitude-Flutter/4.0.0`.

Comment (Mercy811) [reply android/src/main/kotlin/com/amplitude/amplitude_flutter/FlutterLibraryPlugin.kt:12]: It seems to be redundant as the library parameter is a string type value in the primary constructor and is declared as a non-nullable type (String), it cannot be null.

Comment (Mercy811) [reply android/src/main/kotlin/com/amplitude/amplitude_flutter/FlutterLibraryPlugin.kt:12]: `event.library` can be null. I was thinking about `library` of the plugin class.

**Pull request merged**